### DISCUSSION TITLE
Create metadata columns

### DISCUSSION THREAD

**Discussion by celiaccb** [RafaelGB/obsidian-db-folder#506] [Q&A] [answered]
Hi,

One of the options for this plugin is to enable/disable metadata column. 
I am guessing that enabling the modified one (as shown below), should allow me to create a column with the date a row/file was last modified, but I don't know how to create a column that will follow that behaviour.
Thank you.
<img width="810" alt="Screenshot 2022-10-16 at 00 11 57" src="https://user-images.githubusercontent.com/59609780/196011153-37de5c5d-ce57-4c80-b64a-35242d0514ff.png">

**Comment by RafaelGB** [comment, marked as answer]
is correct, enabling that option will show a new column where you can see a date with the last modification once you close the settings menu

**Reply by celiaccb** [reply]
Hi, 
I noticed it does add the column to new databases I create but not to the databases I had created before enabling that option.
Is there a way to add the column to my old databases as well?
Thanks

**Reply by RafaelGB** [reply]
check if __modified__ column already exist inside your markdown ddbb note. maybe they have the hidden flag as true

**Reply by celiaccb** [reply]
The 'show_metadata_modified' line was still set to false (even though I enabled it in the plugin settings), I changed it manually and it works now, thanks!
